resources/tools/index.md

```diff
@@ -22,6 +22,7 @@ permalink: /resources/tools/index.html
                         <p><a href="https://moneroblocks.info">MoneroBlocks</a></p>
                         <p><a href="https://www.exploremonero.com/">Explore Monero</a></p>
                         <p><a href="https://monerovision.com">MoneroVision</a></p>
+                        <p><a href="https://p2pool.io/explorer/">P2Pool</a> (with <a href="http://yucmgsbw7nknw7oi3bkuwudvc657g2xcqahhbjyewazusyytapqo4xid.onion/explorer/">Onion Service</a>)</p>
```

Comment:
The second part of the string needs to be adapted  to be multilingual or it will be displayed only in English.

Reply:
Do I need to add it to all languages in https://github.com/monero-project/monero-site/blob/master/_i18n/ ? I can't translate it all.